DFC.Digital/DFC.Digital.Web.Sitefinity.JobProfileModule.UnitTests/Controllers/JobProfileSearchBoxControllerTests.cs

```diff
@@ -514,7 +514,7 @@ public void SuggestionsTest(string searchTerm, bool useFuzzyAutoCompleteMatching
                     UrlName = "dummy-url"
                 };
 
-                dummySuggestResult.Results = A.CollectionOfDummy<SuggestionResultItem<JobProfileIndex>>(2);
+                dummySuggestResult.Results = A.CollectionOfDummy<SuggestionResultItem<JobProfileIndex>>(2).GroupBy(x => x.MatchedSuggestion).Select(x => x.First());
```

Comment:
Please confirm this how we are grouping on the controller, are we not grouping by label?

Reply:
We are grouping by label as label is holding the matched suggestion

Reply:
To get the distinct value in the dropdown list

Reply:
.GroupBy(x => x.MatchedSuggestion) vs ).GroupBy(x => x.label)